pkg/task/common.go

```diff
@@ -120,8 +122,10 @@ type Config struct {
 	// should be removed after TiDB upgrades the BR dependency.
 	Filter filter.MySQLReplicationRules
 
-	TableFilter        filter.Filter `json:"-" toml:"-"`
-	CheckRequirements  bool          `json:"check-requirements" toml:"check-requirements"`
+	TableFilter       filter.Filter `json:"-" toml:"-"`
+	CheckRequirements bool          `json:"check-requirements" toml:"check-requirements"`
+	// Enabletracing is whether to enable opentracing
```

Comment:
```suggestion
	// EnableTracing is whether to enable opentracing
```

(but this comment looks useless.)